modules/tests/test/create_vm_common_test.go

```diff
@@ -257,6 +264,11 @@ var _ = Describe("Create VM", func() {
 				Expect(err).ShouldNot(HaveOccurred())
 				f.ManageDataVolumes(dataVolume)
 				config.TaskData.SetDVorPVC(fmt.Sprintf("%v:%v", dvWrapper.DiskName, dataVolume.Name), dvWrapper.AttachmentType)
+				// wait for each DV to finish import, otherwise test will fail, because of not finished import of DV
+				Eventually(func() bool {
```

Comment:
we should probably first create all to parallelize and then wait for them to succeed